radars/primary/quadrants/open-edx/rings/hold/blips/tech-radar.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Tech Radar",
+    "ring": "Hold",
+    "quadrant": "Open edX",
+    "isNew": "",
+    "description": "The Tech Radar is a living document that records our technology decisions as they change over time, evaluating their level of adoption and suitability to the Open edX platform’s needs. More information can be found <a href=\"https://openedx.atlassian.net/wiki/spaces/AC/pages/2844786770/Open+edX+Technology+Radar\">here</a>."
```

Comment:
This is good, although I'm not sure I agree with `technology decisions`. Those are represented by OEPs and ADRs. I think this Radar `records our level of adoption of various techniques and technologies as they change over time, evaluating their suitability to the Open edX platform’s needs.`

Reply:
I got that line from the `Why?` section of the [README.md](https://github.com/openedx/openedx-tech-radar/blob/main/README.md). We may want to change that as well.

Reply:
No changes are being made here